schemas/stsci.edu/asdf/transform/ortho_polynomial-1.0.0.yaml

```diff
@@ -0,0 +1,51 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/ortho_polynomial-1.0.0"
+tag: "tag:stsci.edu:asdf/transform/ortho_polynomial-1.0.0"
+title: >
+  Respresents various Orthogonal Polynomial models
+
+description: |
+  A polynomial model represented by its coefficients stored in
+  an ndarray of shape $(n+1)$ for univariate polynomials or $(n+1, n+1)$
+  for polynomials with 2 variables, where $n$ is the highest total degree
+  of the polynomial. The property polynomial_type defines what kind of 
+  polynomial is defined.
+
+  $$P = \sum_{i, j=0}^{i+j=n}c_{ij} * x^{i} * y^{j}$$
+
+  Invertibility: This transform is not automatically invertible.
+
+examples:
+  -
+    - $P = 1.2 + 0.3 * x + 56.1 * x^{2}$
+    - |
+        !transform/ortho_polynomial-1.1.0
```

Comment:
The version of the schema is wrong here.  This should be `ortho_polynomial-1.0.0`.  This is what is causing the schema validation test to fail.